### PR TITLE
Weight based AreaInsert and Dumpable delay, a janitor qol tweak

### PR DESCRIPTION
**Pull request by Krunklehorn**
## About the PR

The delay for dumping and area inserting items into containers now scales with total item weight.

## Why / Balance

Currently, a trash bag full of small items takes ~6 seconds to area insert. A bag full of tiny items takes a whopping 12 and is twice as long when dumping. (24 seconds!) This PR flattens these times and retunes the average to be roughly 5s for all weights in both situations.

Overall, this is a QoL tweak for Janitor players when picking up large piles of ejected cartridges, typically found after firefights.

## Technical details

`StorageComponent` and `DumpableComponent` now sum up the weights of all the items instead of simply counting them. The TimePerItem was turned into a constant and was reduced slightly to retune the average.

`DumpableComponent` reads the same constant to set it's default value. Prototypes with explicitly set TimePerItem fields are unaffected.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Krunk
- tweak: Picking up and dumping trash is less arduous for tiny, plentiful items, such as bullets.

### CODE REVIEW

**Comment by Krunklehorn** [comment]
I don't think anyone is going to complain that their immersion was broken just because a trash bag picked up bullets a little bit quicker.

**Comment by Hanzdegloker** [comment]
I actually find the really long doafter bar very satisfying lol

It's kinda a way to be like "Wow look how long the bar is taking, I'm doing a good job gettin allota trash!" And that stupidly loud sound of dumping it all out brings me dopamine -Jani Enjoyer

**Comment by K-Dynamic** [comment]
> **tl;dr:** buff the janitors too much with "QoL tweaks" like this, and all stations may need to go minus 1 on janitor numbers. And then station becomes dirty too quick and you end up needing more janitor buffs for them to keep up. I don't like the direction of fewer faster janitors. I think the opposite direction is preferable: give janitors **more** to do and keep them busy for longer, so more janitors per station are needed. Higher population of lesser impact janitors helps cushion situations like one of the janitors going SSD and leaving the station crippled. 1/4 janitors going SSD hurts more than 1/8, even if each of those 7 are twice as slow as one of the 3.

Janitor powergaming :trollface: 

God forbid the crew stop making a mess of themselves, we give janitors cleaning grenades, or flood the station with cleanbots. 

QoL is just QoL, nerf janitors directly then rather than making their job clunky, buggy or unintuitive.

**Comment by Krunklehorn** [comment]
Thank you for your input, but I don't think this issue is worth the fuss you're making.

I'm going to leave this for a maintainer to decide.
